app/views/content/steps-to-become-a-teacher/_step_4_find_the_right_training.html.erb

```diff
@@ -0,0 +1,9 @@
+<p>Find out <a href="/ways-to-train">how to train to teach primary or secondary</a>.</p>
+
+<p>You can search for training by where you live, whether you want to study part time and what qualifications you can get.</p>
```

Comment:
Just realised this is quite misleading because it implies by clicking on the link above (to Ways) you'll be able to search by location etc - which you want

Reply:
have done a commit to replace this sentence, so it's more in keeping with the next page in the user journey